plusminus/plusminus.py

```diff
@@ -553,16 +554,16 @@ def __repr__(self):
 
     def __init__(self):
         self.max_number_of_vars = 1000
-        self.max_var_memory = 10 ** 6
```

Comment:
I thought binary operators are supposed to have surrounding spaces.

Reply:
Modified